bigbluebutton-html5/client/main.jsx

```diff
@@ -1,8 +1,16 @@
+/* eslint no-unused-vars: 0 */
```

Comment:
To prevent ESLint from complaining about `React` not being used. We need React in the scope otherwise it will not work.